packages/palette/src/elements/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -370,9 +393,12 @@ const AutocompleteInputDropdown = styled(Box)`
   z-index: 1;
 `
 
-const AutocompleteInputOptions = styled(Box)`
-  /* 308 = Roughly, 5.5 default sized options  */
-  max-height: 308px;
+interface AutocompleteInputOptionProps {
+  maxHeight: string
+}
+
+const AutocompleteInputOptions = styled(Box)<AutocompleteInputOptionProps>`
+  max-height: ${(props) => props.maxHeight};
```

Comment:
This should already accept Box props meaning you could just pass this a `maxHeight` without having to manually interpolate